README.md

```diff
@@ -9,7 +9,32 @@ Copyright 2014, James Roland under Apache License, V2.0. See LICENSE file.
 
 Summary
 -----------
-This project is a .NET implementation of the [Apache Kafka] protocol.  The wire protocol portion is based on the [kafka-python] library writen by [David Arthur] and the general class layout attempts to follow a similar pattern as his project.  To that end, this project builds up from the low level KafkaConnection object for handling async requests to/from the kafka server, all the way up to a higher level Producer/Consumer classes.
+
+This library is a fork of [Jroland's kafka-net library](https://github.com/Jroland/kafka-net), with adjustments and improvements.
+The original project is a .NET implementation of the [Apache Kafka] protocol.  The wire protocol portion is based on the [kafka-python] library writen by [David Arthur] and the general class layout attempts to follow a similar pattern as his project.  To that end, this project builds up from the low level KafkaConnection object for handling async requests to/from the kafka server, all the way up to a higher level Producer/Consumer classes.
+
+##### Improvements and Changes:
+
+- All the code is now async all the way and all blocking operations were removed (except for the high-level Consumer class).
+
+- Bug fixes
+    * When sending messages to the same partition with the same Ack level, order is guaranteed.    
+
+- Tests
+    * Add data-load tests.
+    * Change existing tests to be more consistent.
```

Comment:
After "adjustments and improvements", add: **Note that this library is not interchangeable with kafka-net, as there are breaking changes.**